### PR TITLE
Copy paste is allowed for Execom employees #400

### PR DESCRIPTION
**Pull request by draganakovacevic**
Copy paste is still disabled for test takers, but for Execom employees is allowed.

### CODE REVIEW

**Comment by stanislavgrujic** [comment]
@aberonja @BraneOpacic21 Please review.

File: web/src/main/frontend/src/index.js [hunk 1]
```diff
@@ -24,8 +24,10 @@ import {inviteEmployeeForm} from './components/forms/InviteEmployeeForm';
 
 class Root extends React.Component {
   handleCopyEvent(e) {
-    e.clipboardData.setData('text/plain', 'No cheating! :)');
-    e.preventDefault();
+    if(GuestStore.testId != null && GuestStore.testId != '') {
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/index.js:27]: Not sure what data structures are available for `GuestStore.testId`, but if there are 2 options: `null` and a string value (empty or populated) - then this should probably be enough:

`if(GuestStore.testId) {
// stuff 
}`

or

`if(!GuestStore.testId) {
// stuff 
}`

Depending on the logic you are trying to implement, right?

File: web/src/main/frontend/src/scss/components/_formula.scss [hunk 2]
```diff
@@ -1,4 +1,7 @@
 .formula__block {
+
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/scss/components/_formula.scss:2]: Remove empty rows like these in your CSS/SCSS files - hint: use an extension like prettier or beautify to aid you

File: web/src/main/frontend/src/scss/components/_formula.scss [hunk 3]
```diff
@@ -48,12 +51,13 @@
...
 
   .formula__categories_block {
-    max-height: 240px;
+    position: inherit;
     margin-top: 20px;
-    overflow-y: scroll;
     margin-bottom: 20px;
     width: 100%;
-    z-index: 10;
+    z-index: 15;
```
[the remark this answers is not in the document]
Comment (BraneOpacic21) [reply web/src/main/frontend/src/scss/components/_formula.scss:58]: Further explanation: Skipping numbers would potentially lead to a bigger problem once the project scales, leaving you frustrated trying to 'debug' CSS

File: web/src/main/frontend/src/scss/components/_formula.scss [hunk 4]
```diff
@@ -48,12 +51,13 @@
...
+    display: inline-block;
+    flex: 1;
```
Comment (BraneOpacic21) [inline web/src/main/frontend/src/scss/components/_formula.scss:60]: /n

Separate your CSS rules, please

**Pull request merged**